boards/arm/nucleo_f103rb/doc/nucleof103rb.rst

```diff
@@ -124,33 +128,26 @@ This interface is supported by the openocd version included in the Zephyr SDK.
 Flashing an application to Nucleo F103RB
 ----------------------------------------
 
-The sample application :ref:`blinky-sample` is being used in this tutorial.
-
-.. code-block:: console
-
-   $ cd <zephyr_root_path>
-   $ source zephyr-env.sh
-   $ cd $ZEPHYR_BASE/samples/basic/blinky
-   $ make BOARD=nucleo_f103rb
-
-Connect the Nucleo F103RB to your host computer using the USB port.
-Then, enter the following command:
+Here is an example for the :ref:`blinky-sample` application.
 
-.. code-block:: console
-
-   $ make BOARD=nucleo_f103rb flash
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: nucleof103rb
```

Comment:
nucleo_f103rb

Reply:
Thanks for catching that. I was just copying the file name in as I assumed that was the convention after they stopped just being called "board.rst"